Sources/ClientRuntime/Serialization/SerializationUtils/XML/ValueReaders.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import class SmithyXML.Reader
+
+extension Reader {
+
+    public func read() throws -> ByteStream? {
```

Comment:
This does not happen in practice, but is needed to provide a `.read()` implementation for `ByteStream`, which may be a member on output models.